src/Examples/SequenceToSequence.cs

```diff
@@ -26,7 +26,11 @@ namespace TorchSharp.Examples
     public class SequenceToSequence
     {
         // This path assumes that you're running this on Windows.
+#if NETFX472
```

Comment:
```suggestion
#if NET472_OR_GREATER
```
Same for all of the rest of places where you are using this constant.